ide/static/js/layer.js

```diff
@@ -90,6 +90,7 @@ class Layer extends React.Component {
         }}
         data-type={this.props.type}
         onClick={(event) => this.props.click(event, this.props.id)}
+        onContextMenu={(event) => this.props.onContextMenu(event, this.props.id)}
```

Comment:
Webpack error:
`93:46  error  'onContextMenu' is missing in props validation  react/prop-types`